mindsdb/api/http/namespaces/agents.py

```diff
@@ -273,6 +273,8 @@ def post(self, project_name, agent_name):
             )
 
         # Add OpenAI API key to agent params if not already present.
+        if not existing_agent.params:
```

Comment:
it does not matter, but maybe `existing_agent.params is None`? because `not {}` is true